meetings/2016-11-23.md

```diff
@@ -0,0 +1,272 @@
+* Rich Trott @Trott (CTC)
+  * Onboarded @italoacasas
+  * Fixing flaky tests (test-inspector flakiness fix should land today)
+  * PR and issue review
+  * Preparing for Code & Learn
+
+
+---
+
+## Agenda
+
+### nodejs/node
+
+* Revert "buffer: runtime deprecation of calling Buffer without new"
+  [#9529](https://github.com/nodejs/node/pull/9529)
+
+
+---
+
+## Previous Meeting Review
+
+* A new New CTC Meeting Schedule Proposal 
+  [#29](https://github.com/nodejs/CTC/issues/29)
+* Documentation WG Status [#32](https://github.com/nodejs/CTC/issues/32)
+* Revert "buffer: runtime deprecation of calling Buffer without new"
+  [#9529](https://github.com/nodejs/node/pull/9529)
+  * Moved to private discussion.
+* tls: fix leak of WriteWrap+TLSWrap
+  combination[#9626](https://github.com/nodejs/node/pull/9626)
+  * **Next steps**: Fix in next v7.x release than backport.
+* src: don't call into VM from AsyncWrap
+  destructor[#9467](https://github.com/nodejs/node/pull/9467)
+  * **Next steps**: @trevnorris implement steps described in previous minutes: start by removing
+    weak handles from Node.
+
+
+---
+
+## Minutes
+
+### Revert "buffer: runtime deprecation of calling Buffer without new" [#9529](https://github.com/nodejs/node/pull/9529)
+
+See also <https://github.com/nodejs/node/issues/9531> for broader discussion.
+
+@addaleax: Most discussion this week has been about general way forward rather
+than this specific PR.
+
+Does anyone think the deprecation warning introduced in v7.x serves its purpose?
+
+@ChALkeR: It probably doesn’t. From a security point-of-view it doesn’t. It had
+another purpose - provide a way to inherit from Buffer as a class, and to that
+it has been successful.
+
+@mhdawson: Does anyone object to removing the deprecation warning?
+
+@ChALkeR: If we decide to remove use of Buffer (with new and without), will we
+in retrospect still consider removal of the deprecation warning a good move?
+
+@mhdawson: We don’t have a good answer, and we may have to reintroduce the
+deprecation in v8.x. But that shouldn’t change our action today.
+
+@Fishrock123: If we revert this we shouldn’t necessarily deprecate anything in
+v8.x.
+
+@ChALkeR: Then reversion would be a good next step.
+
+@mhdawson: “We put this in, there’s been some negative impact. So we’ll remove
+it for now and reconsider the path forward.”
+
+@addaleax: We should set aside the discussion about subclassability and
+inheritance for now. It would require removing `Buffer` without new.
+
+It’s become obvious that we’ll never be able to deprecate `Buffer` without new.
+We may need a new API to support subclassability, it wouldn’t be too difficult.
+
+@trott: Perhaps the CTC decision-making process is not effective in dealing with
+this issue. Perhaps we should delegate to a small subset of CTC and non-CTC
+people who have strong opinions and interest and can close on a recommendation.
+
+@mhdawson: At least we have consensus to revert the deprecation.
```

Comment:
should be:
@mhdawson:  I don't think I've heard objections in our discussion today to reverting the deprecation.